content/en/docs/reference/command-line-tools-reference/feature-gates.md

```diff
@@ -134,6 +134,7 @@ For a reference to old feature gates that are removed, please refer to
 | `KubeletPodResourcesGetAllocatable` | `true` | Beta | 1.23 | |
 | `KubeletTracing` | `false` | Alpha | 1.25 | |
 | `LegacyServiceAccountTokenNoAutoGeneration` | `true` | Beta | 1.24 | |
```

Comment:
Pls remove line 136 since the feature is now in the 'GA/deprecated' group.

Reply:
done